astroquery/cadc/core.py

```diff
@@ -544,4 +546,4 @@ def get_access_url(service, capability=None):
                        "anonymous or cookie access".format(capability))
 
 
-Cadc = CadcClass
+Cadc = CadcClass()
```

Comment:
I guess this may be the culprit.

Reply:
it is; I'm working on it but there are deep things beneath this unfortunately....